### PR TITLE
HOMS-102 Add tests for select

### PR DESCRIPTION
**Pull request by GideonEXE**

### CODE REVIEW

File: spec/hbw/features/select_spec.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+feature 'Check select with', js: true do
+  let(:placeholder)  { 'placeholder' }
+  let(:first_value)  { 'Option 1' }
+
+  $start_number = 3
```
Comment (flash-gordon) [inline spec/hbw/features/select_spec.rb]: do not use global variables in application's code. ever

**Comment by GideonEXE** [comment]
UPD

File: hbw/app/models/hbw/fields/select.rb [hunk 2]
```diff
@@ -71,7 +71,7 @@ def options_condition
       end
 
       def placeholder
-        definition.fetch('placeholder', '')
+        definition.fetch('placeholder', nil)
```
Comment (flash-gordon) [inline hbw/app/models/hbw/fields/select.rb]: if definition is a hash, this makes no sense, just use `[]` `definition['placeholder']`

**Comment by GideonEXE** [comment]
UPD

**Comment by GideonEXE** [comment]
UPD

**Pull request merged**